bpfd/src/bpf.rs

```diff
@@ -196,8 +196,14 @@ impl BpfManager {
             // TODO: Should probably check for pinned prog on bpffs rather than assuming they are attached
             program.set_attached();
             debug!("rebuilding state for program {}", id);
-            self.rebuild_map_entry(id, program.data()?.map_owner_id());
+            let map_owner_id = match program.data() {
```

Comment:
why do we need this map can't you just move

`self.rebuild_map_entry(id, program.data()?.map_owner_id());`

below the insert?

Additionally this is now ignoring the error is that what we want?

Reply:
Then just rebuild the map entry before inserting into programs right? That __should__ allow you to get past the borrow errors

Reply:
Ok, done